pennylane_lightning_kokkos/src/simulator/MeasuresFunctors.hpp

```diff
@@ -58,25 +58,25 @@ template <class Precision> struct getCDFFunctor {
 /**
  *@brief Sampling using Random_XorShift64_Pool
  *
- * @param samples_ Kokkos::View of the generated samples.
- * @param cdf_  Kokkos::View of cumulative probability distribution.
+ * @param samples_ ::Kokkos::View of the generated samples.
```

Comment:
Do we want to have operator:: in the docstrings?

Reply:
Nah! Fixing this too.